__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("status 200,returns an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics[0]).toHaveProperty("slug");
```

Comment:
This only checks the first item in your topics array. What if there are any problems with the rest of your array? Remember, you can iterate over data in a test :)